spec/src/legacy_app_spec.js

```diff
@@ -20,4 +20,12 @@ describe('LegacyApp', () => {
       expect(app.switchTo).toHaveBeenCalledWith('main', data.user);
     });
   });
+
+  describe('#logClosedApp', () => {
+    it('is called when the app is exiting', function() {
+      spyOn(app, 'logClosedApp');
```

Comment:
Sorry I realise now my first comment was misleading. We should probably leave this file alone and create another spec for Base.